lib/rojo-thinks.js

```diff
@@ -0,0 +1,73 @@
+const rojoThinks = (question) => {
+  const startWithWhen = /when.*/i;
+  const startWithWhat = /what.*/i;
+  const startWithWhere = /where.*/i;
+  const startWithWhich = /which.*/i;
+
+  const containsTime = /.+time.*/i;
+  const containsAge = /.+age.*/i;
+  const containsDay = /.+day.*/i;
+  const containsGo = /.+go.*/i;
+  const containsMy = /.+my.*/i;
+  const containsCountry = /.+country.*/i;
+  const containsCity = /.+city.*/i;
+
+  switch (true) {
+    // test when
+    case startWithWhen.test(question):
+      switch (true) {
+        // test when
+        default:
+          return 'Let/s say 12pm.';
+      }
+    // test what
+    case startWithWhat.test(question):
+      switch (true) {
+        // test what & time
+        case containsTime.test(question):
+          return '4:30pm as I recall.';
+          break;
```

Comment:
A `return` statement returns control to whatever called a function, so the `break` statement can actually never be reached if it follows `return`.